app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -16,7 +16,7 @@ def process
 
       # Get the fields that we're going to pull over
       @item = PortfolioItem.create!(generate_attributes)
-      @item.icons << create_icon(@service_offering.service_offering_icon_id)
+      @item.icons << create_icon(@service_offering.service_offering_icon_id) if @service_offering.service_offering_icon_id.present?
```

Comment:
LGTM.  Are there tests we can wrap around this?

Reply:
Added a test to make sure it handles when there is no icon appropriately.